dev-schema/attribute/nationality.json

```diff
@@ -14,7 +14,7 @@
 		"denonym": {
 			"type": "string",
 			"title": "Denonym",
-			"description": "The word used to identify citizens of the country"
+			"description": "Enter the word."
```

Comment:
This doesn't look right

Reply:
Let me check with Sasha

Reply:
it should still be a meaningful description, although a shorter one

Reply:
I've updated it, please review.